dev/codeserver/java/com/google/gwt/dev/codeserver/dev_mode_on.js

```diff
@@ -122,19 +122,22 @@
     result.style.background = '#ddd';
     result.style.border = '2px outset #ddd';
     result.style.padding = '3pt';
-    result.setAttribute('href', 'javascript:' + encodeURIComponent(javascript));
+    result.onclick = javascript;
     result.title = 'Tip: drag this button to the bookmark bar';
     return result;
   }
 
   function makeCompileBookmarklet(codeserver_url, module_name) {
     var bookmarklets_js = codeserver_url + 'dev_mode_on.js';
-    var javascript = '{ window.__gwt_bookmarklet_params = {'
-        + 'server_url:\'' + codeserver_url + '\','
-        + 'module_name:\'' + module_name + '\'};'
-        + ' var s = document.createElement(\'script\');'
-        + ' s.src = \'' + bookmarklets_js + '\';'
-        + ' void(document.getElementsByTagName(\'head\')[0].appendChild(s));}';
+    var javascript = function() { 
+	  window.__gwt_bookmarklet_params = {
+        server_url: codeserver_url,
+        module_name: module_name
+	  };
+       var s = document.createElement('script');
+       s.src = bookmarklets_js;
+       void(document.getElementsByTagName('head')[0].appendChild(s));
+	};
```

Comment:
Indentation is kind of all over the place here, can you make it consistent?

Reply:
Fixed now